### PR TITLE
DDF-1192: Removed unused dependencies to reduce the number of OWASP vulnerabilities.

### PR DESCRIPTION
**Pull request by lessarderic**
[Review on Reviewable](https://reviewable.io/reviews/codice/ddf-ui/76)

### CODE REVIEW

**Comment by lessarderic** [comment]
@shaundmorris, @stustison

File: search-ui/standard/pom.xml [hunk 1]
```diff
@@ -92,26 +69,6 @@
...
-            <version>2.14.2</version>
-        </dependency>
-        <dependency>
-            <groupId>ddf.security.core</groupId>
-            <artifactId>security-core-api</artifactId>
-            <version>2.3.0</version>
-        </dependency>
-        <dependency>
-            <groupId>ddf.platform.error</groupId>
-            <artifactId>platform-error-servlet</artifactId>
```
Comment (stustison) [inline search-ui/standard/pom.xml:112]: I don't think you can actually remove this error servlet. It is embedded into this bundle.

Comment (lessarderic) [inline search-ui/standard/pom.xml:112]: Added platform-error-servlet back. How about the other ones?

**Comment by pklinef** [comment]
:+1:

File: search-ui/standard/pom.xml (hunk 1 above)
Comment (stustison) [inline search-ui/standard/pom.xml:112]: The other ones are embedded as well. Did you try running this to make sure that the map comes up as expected etc.

**Comment by shaundmorris** [comment]
Verified build, integration tests, and basic search UI capabilities were still working.

File: search-ui/standard/pom.xml (hunk 1 above)
Comment (shaundmorris) [inline search-ui/standard/pom.xml:112]: I verified the map was coming up as expected on the latest.

**Comment by lessarderic** [comment]
Thanks for verifying this @shaundmorris. I did verify it too before committing and did work for me as well.

**Comment by stustison** [comment]
Can you also set up the context policy so that hitting the page will generate an error and ensure that the correct error page displays.

**Pull request merged**